src/Sylius/Bundle/ApiBundle/Resources/config/api_resources/Address.xml

```diff
@@ -12,7 +12,7 @@
            xsi:schemaLocation="https://api-platform.com/schema/metadata https://api-platform.com/schema/metadata/metadata-2.0.xsd"
 >
     <resource class="%sylius.model.address.class%" shortName="Address">
-        <attribute name="route_prefix">shop</attribute>
+        <attribute name="route_prefix">shop/account</attribute>
```

Comment:
I'm not sure about this change. Do we serialize the whole address to orders or do we have IRI to fetch it on your own?

Reply:
`src/Sylius/Bundle/ApiBundle/Resources/config/serialization/Address.xml` in this file we are serializing all properties from the address in `checkout` and `orderShow`